src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': todo.completed,
+        },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+      {todo.user && (
+        <UserInfo user={todo.user} />
+      )}
```

Comment:
Make a destructuring from todo
```suggestion
      className={classNames('TodoInfo', {
          'TodoInfo--completed': todo.completed,
        })}
    >
      <h2 className="TodoInfo__title">
        {todo.title}
      </h2>
      
      {todo.user && (
        <UserInfo user={todo.user} />
      )}
```